### PR TITLE
Describe new metrics provided by VNS3 integration, update pointers, modernize language, add example dashboard

### PR DESCRIPTION
**Pull request by bob-smetana**
### What does this PR do?

Brings the description of the VNS3 integration up to reflect current reality.

### Motivation

A new version of the datadog-agent plugin for VNS3 has been released, adding new reported metrics and other functionality.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [x] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

n/a

### CODE REVIEW

**Comment by bob-smetana** [comment]
Third time's the charm!

File: vns3/README.md
Comment (brett0000FF) [inline vns3/README.md:35]:
```suggestion
Deploy and configure the Cohesive Networks Datadog plugin according to the [DataDog Agent Plugin Details
][7] documentation.
```

**Review by brett0000FF — APPROVED** [review]
Thanks, looks good! I just left a small suggestion to make one of the links more descriptive.

**Comment by bob-smetana** [comment]
Actually I may have an issue with the dashboard.json in here.
Guess regular support will be the best resource for that?

**Comment by bob-smetana** [comment]
Support was very helpful as always - one of the widgets on that dashboard was somehow tainted. JSON file imports correctly after recreating that widget.
I am satisfied that this is ready for merge upon final approval.

**Comment by bob-smetana** [comment]
Not to nag, just making sure this hasn't dropped off the radar.
Still happy with latest commit, ready for a merge.

**Review by brett0000FF — APPROVED** [review]

**Comment by brett0000FF** [comment]
@bob-smetana - Thanks for this and apologies for the delay! We still need a couple required approvals before merging. Let me ping them to make sure it's on their radar.

CC @DataDog/agent-integrations @DataDog/reporting-and-sharing

**Review by steveny91 — APPROVED** [review]

File: vns3/CHANGELOG.md
Comment (steveny91) [inline vns3/CHANGELOG.md:1]: Hmm not sure why this is listed as Nomad

Comment (bob-smetana) [reply vns3/CHANGELOG.md:1]: Nor am I - not even sure what it is referring to. Don't think I put it there. Is it an issue?

Comment (steveny91) [reply vns3/CHANGELOG.md:1]:
```suggestion
# CHANGELOG - VNS3
```
Non blocking nit. I guess just some good housekeeping.

**Comment by bob-smetana** [comment]
Polite reminder that this is ready for merge

**Comment by brett0000FF** [comment]
@steveny91 - Could you please merge this one for @bob-smetana ? Docs team doesn't usually merge these. Thanks!

**Pull request merged**
